android-components/components/support/base/src/main/java/mozilla/components/support/base/log/sink/TestModeLogSink.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.base.log.sink
+
+import mozilla.components.support.base.log.Log
+
+/**
+ * <code>LogSink</code> implementation that prints to console.
```

Comment:
Wondering if we would be better served doing `[LogSink]` since I know that will allow you to jump to the code, but I am not sure if `<code>` will do that.

Reply:
Good call, updated back to to [LogSink]